youtube_dl/extractor/nhk.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class NhkVodIE(InfoExtractor):
+    _VALID_URL = r'http://www3.nhk.or.jp/nhkworld/en/vod/(?P<id>.+)\.html'
+    # _TESTS = []  # TODO videos I've found are all available only for a limited period of time. That automatically makes such test cases obsolete. :/
```

Comment:
If you can't find any permanent videos, add a temporary video and `'skip_download'` to `True`. This at least tests the `_VALID_URL`.